src/sentry/models/groupowner.py

```diff
@@ -111,7 +111,7 @@ def get_autoassigned_owner(cls, group_id, project_id, autoassignment_types):
                 group_id=group_id, project_id=project_id, type__in=autoassignment_types
             )
             .exclude(user_id__isnull=True, team_id__isnull=True)
-            .order_by("type")
+            .order_by("type", "date_added")
```

Comment:
Question: In theory, wouldn't we want to assign the _newest_ group owner? That would be the one most recently created by an ownership rule, and since those are processed in order and the bottom-most rule takes precedent?

Reply:
that is wild, great catch 😖 

Might be worth calling out in a comment as well.